notebooks/query_uploaded_database.ipynb

```diff
@@ -0,0 +1,648 @@
+{
```

Comment:
<p>I think this can be removed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nasaharvest/street2sat/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>